workflows/diagnostics/fv3net/diagnostics/offline/views/create_report.py

```diff
@@ -106,6 +107,14 @@ def _create_arg_parser() -> argparse.Namespace:
         default=None,
         help=("Training data configuration yaml file to insert into report"),
     )
+    parser.add_argument(
+        "--wandb",
```

Comment:
It's a bit confusing to have the argo workflow flip the default. I'd propose having wandb on by default everywhere...e.g. jeremy doesn't use argo.
```suggestion
        "--no-wandb",
```

Reply:
It also looks like you have a test failure related to this:
```
E       AttributeError: 'CreateReportArgs' object has no attribute 'wandb'
```

`CreateReportArgs` is a mock . Can you you refactor the tests to use the real argument parser to create `args`, so that we don't have to worry about the mock getting out of sync again. Either that or re-target the tests at a slightly lower level API that doesn't refer to `args`, but rather the fields in `args`. e.g. `run_report(args.url, args.other_needed_arg, args.output_url)`. I'm not sure if such an api exists, would be nice if it did to allow decoupling the argparse and actual report running logic.